qute.ls/com.redhat.qute.ls/src/main/java/com/redhat/qute/parser/template/sections/InputSection.java

```diff
@@ -0,0 +1,44 @@
+/*******************************************************************************
+* Copyright (c) 2023 Red Hat Inc. and others.
+* All rights reserved. This program and the accompanying materials
+* which accompanies this distribution, and is available at
+* http://www.eclipse.org/legal/epl-v20.html
+*
+* SPDX-License-Identifier: EPL-2.0
+*
+* Contributors:
+*     Red Hat Inc. - initial API and implementation
+*******************************************************************************/
+package com.redhat.qute.parser.template.sections;
+
+import com.redhat.qute.parser.template.ASTVisitor;
+import com.redhat.qute.parser.template.Section;
+import com.redhat.qute.parser.template.SectionKind;
+
+/**
+ * Input section.
+ * 
+ */
+public class InputSection extends Section {
```

Comment:
At first `<input` is an html element and not a qute section. A qute section starts with `{#`. But I think you don't need this class, right? If yes, please remove it.